blog/models.py

```diff
@@ -39,6 +39,7 @@ class Article(db.Document):
     author = db.ReferenceField(User)
     language = db.StringField(min_length=2, max_length=2, default='en')
     images = db.ListField()
+    is_published = db.BooleanField()
```

Comment:
I guess we could use a `publication_date` instead of a `is_published`.

`is_published` could then be a property:

```python
@property
def is_published():
    return bool(self.publication_date)
```